docs/usage/install.rst

```diff
@@ -1,3 +1,230 @@
-Installation
-============
+Installation Instructions
+#########################
+
+PlaidML supports :ref:`Ubuntu`, :ref:`macOS` , and :ref:`Microsoft Windows` operating systems.
+
+Ubuntu
```

Comment:
Can we use tabs for these installation instructions now that we have them?

Reply:
Done